pkgs/by-name/ff/ffmpeg-livepeer/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  ffmpeg_7-headless,
+  lib,
+  fetchFromGitHub,
+  apple-sdk_11,
+  stdenv,
+  clang,
+}:
+
+let
+  version = "7.0.1-unstable-2024-07-10";
+in
+
+(ffmpeg_7-headless.override {
+  inherit version;
+  withCudaLLVM = true;
+  source = fetchFromGitHub {
+    owner = "livepeer";
+    repo = "FFmpeg";
+    rev = "d9751c73e714b01b363483db358b1ea8022c9bea"; # From branch n*-livepeer
+    hash = "sha256-IJVpb/k+obGFD9uOoIVHCd2ZiGL3CA4CV3D+Q9vMbQM=";
+  };
+}).overrideAttrs
+  (old: {
+    pname = "ffmpeg-livepeer";
+    buildInputs =
+      old.buildInputs or [ ]
+      ++ lib.optionals stdenv.hostPlatform.isDarwin [
+        apple-sdk_11
+      ];
```

Comment:
This will becomme unnescessary after the current staging-next gets merged

Reply:
The current staging-next is for 24.11, which still has the 10.12 SDK as the default on x86_64-darwin. The work to make the 11.3 SDK the default is still in progress and hasn’t been merged yet.

Reply:
Should we remove this and let this PR target staging-next then?

Reply:
According to release schedule (#339153) `staging-next` is planned to be merged into master on the 12th of November

Reply:
Or, yes, just target this at `staging-next` directly and then we can merge it.